lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -71,8 +72,13 @@ def failure_message
         red_name = ColorizedString.new(c.name).red
         "> #{red_name}: #{c.doom_reason}"
       end
-      intro = "The following containers are in a state that is unlikely to be recoverable:\n"
-      intro + container_messages.join("\n") + "\n"
+
+      intro = if unmanaged?
+        "The following containers encountered errors:"
```

Comment:
The "unlikely to be recoverable" message doesn't really make sense for a dead run-once pod.